tools/build-tools/src/repoPolicyCheck/handlers/lockfiles.ts

```diff
@@ -13,7 +13,7 @@ const urlPattern = /(https?[^"@]+)(\/@.+|\/[^/]+\/-\/.+tgz)/g;
 
 export const handler: Handler = {
     name: "package-lockfiles",
-    match: /^.*?package-lock\.json$/i,
+    match: /^.*?[^_]package-lock\.json$/i, // Ignore _package-lock.json
```

Comment:
nit: a bit more context in the comment would be helpful.

Reply:
I'm going to leave it mysterious and let folks use blame to go back to see this PR / linked PR. Maybe other use cases will arise too.